test/novu/events_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule Novu.EventsTest do
-  use ExUnit.Case, async: true
```

Comment:
We were asynchronously setting global Novu domain config with `Application.put_env/3` to use `bypass.port`, which caused tests to begin clashing with each other and failing.

Reply:
Async is hard :facepalm: